src/test/java/ancaMarian/tests/tema6/HerokuForms.java

```diff
@@ -0,0 +1,144 @@
+package ancaMarian.tests.tema6;
+
+import ancaMarian.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import teofilursan.driver.BrowserManager;
+
+public class HerokuForms {
+
+    static ChromeDriver driver = null;
+
+    public static void main(String[] args) throws InterruptedException {
+        navigateToHerokuHomePage();
+        maximizeWindow(driver);
+        navigateToHerokuFormsPage();
+        inputUsername();
+        inputPassword();
+        inputComment();
+        addFile();
+        selectCheckbox();
+        deselectCheckbox();
+        scrollToTheBottomOfThePage();
+        selectRadioItem();
+        selectFromMultipleValues();
+        selectFromDropdownList();
+        scrollToSubmitButton();
+        FileUtils.takeScreenshot(driver, "form");
+        goBackToFormPage();
+        try {
+            goBackToFormPage();
+        } catch (NoSuchElementException e) {
+            FileUtils.takeScreenshot(driver, "ElementNotFound");
+            System.out.println("Nu s-a gasit elementul pe pagina. S-a salvat un screenshot cu denumirea ElementNotFound.");
+        } finally {
+            closeBrowser();}
+    }
+
+    public static void navigateToHerokuHomePage(){
+        driver = ancaMarian.driver.BrowserManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/index.html");
+        System.out.println("Am deschis Heroku index page.");
+    }
+
+    public static void maximizeWindow(ChromeDriver driver){
+        driver.manage().window().maximize();
+    }
+
+    public static void navigateToHerokuFormsPage() {
+        WebElement formsPage = driver.findElement(By.id("htmlformtest"));
+        formsPage.click();
+        System.out.println("Am deschis Heroku Forms page.");
+    }
+
+    public static void inputUsername(){
+        WebElement username = driver.findElement(By.name("username"));
+        username.sendKeys("Username1");
+        System.out.println("Am adaugat un username.");
+    }
+
+    public static void inputPassword(){
+        WebElement password = driver.findElement(By.name("password"));
+        password.sendKeys("password1");
+        System.out.println("Am adaugat o parola.");
+    }
+
+    public static void inputComment(){
+        WebElement comment = driver.findElement(By.name("comments"));
+        comment.clear();
+        comment.sendKeys("comment1");
+        System.out.println("Am adaugat un comentariu.");
+    }
+
+    public static void addFile(){
+        WebElement uploadFile = driver.findElement(By.name("filename"));
+        uploadFile.sendKeys("/Users/ancapatriciamarian/Desktop/procyon/src/text.txt");
```

Comment:
Try to avoid using local paths when adding files. This won't work on other machines. Create the file inside the repo, in your package for homework 6 and get the path from there. Thanks!
@tincageorgianaalina  can help you